src/mark.ts

```diff
@@ -79,6 +79,26 @@ export interface MarkConfig extends VgMarkConfig {
    * __Note:__ This property cannot be used in a [style config](mark.html#style-config).
    */
   color?: string;
+
+  /**
+   * Offset for x channel in pixels. The offset is applied before rotation by the _angle_ property.
```

Comment:
Angle isn't supported for all marks, so I would just cut the part about angle to avoid confusion.